### PR TITLE
Bugfix/display macros based on user role

### PR DESCRIPTION
**Pull request by darshanitaliya7110**
### What this does
Ticket details - when macros list is empty and user is not admin/owner hide this

### Implementation
display macros based on user role

### Testing
Member:
image
Admin/Owner
image

### CODE REVIEW

File: app/details/[ticket_id]/ticketDetails.tsx [hunk 1]
```diff
@@ -707,31 +707,36 @@ function TicketDetails(props: Props) {
...
-                          iconViewBox={''}
-                          style={{ bottom: 60, maxWidth: 146, width: '100%' }}
-                          isMacro={
-                            currentWorkspace?.role === UserRole.OWNER ||
-                            currentWorkspace?.role === UserRole.ADMIN
-                          }
-                        />
-                      )}
-                    </div>
+                    {macros.length > 0 && (
```
Comment (aniket-pixer) [inline app/details/[ticket_id]/ticketDetails.tsx:710]: Here condition will be like
{(currentWorkspace?.role === UserRole.OWNER ||
                          currentWorkspace?.role === UserRole.ADMIN || macros.length > 0 && (

File: app/details/[ticket_id]/ticketDetails.tsx [hunk 2]
```diff
@@ -707,31 +707,36 @@ function TicketDetails(props: Props) {
...
+                      <div className='tag-div'>
+                        {(currentWorkspace?.role === UserRole.OWNER ||
```
Comment (aniket-pixer) [inline app/details/[ticket_id]/ticketDetails.tsx:712]: Remove condition from here!

File: app/details/[ticket_id]/ticketDetails.tsx (hunk 1 above)
Comment (darshanitaliya7110) [reply app/details/[ticket_id]/ticketDetails.tsx]: Done

File: app/details/[ticket_id]/ticketDetails.tsx (hunk 2 above)
Comment (darshanitaliya7110) [reply app/details/[ticket_id]/ticketDetails.tsx]: Done

**Pull request merged**